src/igvfd/schemas/sample_ontology_term.json

```diff
@@ -0,0 +1,64 @@
+{
+    "title": "Sample ontology term",
+    "description": "Schema for an ontology term.",
+    "$id": "/profiles/sample_ontology_term.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "term_id",
+        "term_name"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "ontology_term.json#/properties"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "term_id": {
+            "@type": "@id",
+            "title": "Ontology ID",
+            "description": "An ontology identifier describing a biological sample, assay, trait, or disease.",
```

Comment:
This description seems like it would be on the level of ontology not at the level of sample_ontology

Reply:
Will fix this.